cmd/hypper/testdata/testcharts/hypper-annot/Chart.yaml

```diff
@@ -10,3 +10,10 @@ annotations:
   hypper.cattle.io/release-name: my-hypper-name
   catalog.cattle.io/namespace: fleet-system
   catalog.cattle.io/release-name: fleet
+  hypper.cattle.io/shared-dependencies: |
```

Comment:
can we create a different chart for this so we can manipulate tests without affecting other tests?

Reply:
Mmh, I'm not sold on that. Thing is, by adding this annotations to that chart, we just made sure that the previous tests, which pass unchanged, keep working with the `shared-dependencies` annotation. That way we don't need to create additional tests for now. Is that convincing enough? :>

In the future we may need additional testcharts for testing install of chart+shared-deps, though.

Reply:
its just that if we now extend the install tests for example and modify this chart (I think we use it there) we could potentially be breaking tests for shared-deps by manipulating it.

In any case we can cross that bridge when we reach it :)